File: program (113).c
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//  Function Name  :    Display
//  Description :       Accept 'N' number from user and display all Characters till the End of the Alphabet.. 
//  Input :             char 
//  Output :            char
//  Author :            Aman Shaikh    
//  Date:               19/11/2024
// 
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>

void Display(char cVal)
{
        int i = 0;

        if(cVal >= 65 && cVal <= 90)
        {
            for(i = cVal; i <= 90; i++)
            {
                printf("%c\t",i);
            }
        } 
        else if(cVal >= 97 && cVal <= 122)
        {
            for(i = cVal; i >= 97; i--)
            {
                printf("%c\t",i);
            }
        }
        else
        {
            return;
        }
}

int main()
{
    char cValue = '\0';

    printf("Enter the character : ");
    scanf("%c",&cValue);   

    Display(cValue);

    return 0;
}

File: program (113).java
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//  Function Name  :    CountDiff
//  Description :       Return the Difference between summation of even elements and odd elements. 
//  Input :             int 
//  Output :            int
//  Author :            Aman Shaikh    
//  Date:               14/12/2024 
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Digit
{
    public int CountDiff(int iNo)
    {
        int i = 0, iCnt = 0, iCount = 0;
        while(iNo != 0)
        {
            i = iNo % 10;
            if(i%2 != 0 && i != 0)
            {
                iCnt = iCnt + i;
            }
            else if(i%2 == 0 && i != 0)
            {
                iCount = iCount + i;
            } 
            iNo = iNo / 10;
        }
        return iCount-iCnt;
        
    }
}

class Assignment38E
{
    public static void main(String Arr[])
    {
        int iValue = 0;
        
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter number : ");
        iValue = sobj.nextInt();  

        Digit aobj = new Digit();
        int iResult = aobj.CountDiff(iValue); 

        System.out.println("Difference is : "+iResult);
    }
}